shell/gpu/gpu_surface_gl_skia.cc

```diff
@@ -250,9 +249,9 @@ std::unique_ptr<SurfaceFrame> GPUSurfaceGLSkia::AcquireFrame(
       };
 
   framebuffer_info = delegate_->GLContextFramebufferInfo();
-  // Partial repaint is enabled by default
-  framebuffer_info.supports_partial_repaint = supports_partial_repaint_;
-  framebuffer_info.existing_damage = existing_damage_;
+  if (!framebuffer_info.existing_damage.has_value()) {
+    framebuffer_info.existing_damage = existing_damage_;
+  }
```

Comment:
This looks good to me! It should work given that FBOResetAfterPresent is not always called and it should also work if FBO was indeed reset and the Embedder passed information about the existing damage.